src/classrooms/classrooms.controller.ts

```diff
@@ -27,6 +27,11 @@ import { ClassroomsService } from "./classrooms.service";
 export class ClassroomsController {
   constructor(private readonly classroomsService: ClassroomsService) {}
 
+  @Get(":id")
+  getClassroom(@Param("id") id: string) {
+    return this.classroomsService.getClassroomById(+id);
```

Comment:
```suggestion
    return this.classroomsService.getClassroomById(Number(id));
```

Let's try to be clear with what we are trying to do